### ISSUE TITLE
How much data should be used for my own dataset?

### ISSUE THREAD

**Issue by Zakeiswo** [yysijie/st-gcn#119]
Hello
  Thanks for your work. I have used my own dataset to train. The dataset has 10 classes, and 60 samples for one class. However the accuracy is very low,31% for top1 and 62% for top5. So I want to ask about how many samples for one class is better?

**Comment by yjxiong** [comment]
600 samples in total seems a little too few. But you can try finetuning from one of the pretrained models provided.
